test/tests/mysql-skip-tzinfo/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+export MYSQL_ROOT_PASSWORD='this is an example test password'
+export MYSQL_USER='0123456789012345' # "ERROR: 1470  String 'my cool mysql user' is too long for user name (should be no longer than 16)"
+export MYSQL_PASSWORD='my cool mysql password'
+export MYSQL_DATABASE='my cool mysql database'
+
+cname="mysql-container-$RANDOM-$RANDOM"
+cid="$(
+	docker run -d \
+		-e MYSQL_INITDB_SKIP_TZINFO=1 \
+		-e MYSQL_ROOT_PASSWORD \
+		-e MYSQL_USER \
+		-e MYSQL_PASSWORD \
+		-e MYSQL_DATABASE \
+		--name "$cname" \
+		"$image"
+)"
+trap "docker rm -vf $cid > /dev/null" EXIT
+
+mysql() {
+	docker run --rm -i \
+		--link "$cname":mysql \
+		--entrypoint mysql \
+		-e MYSQL_PWD="$MYSQL_PASSWORD" \
+		"$image" \
+		-hmysql \
+		-u"$MYSQL_USER" \
+		--silent \
+		"$@" \
+		"$MYSQL_DATABASE"
+}
+
+. "$dir/../../retry.sh" --tries 20 "echo 'SELECT 1' | mysql"
+
+echo "SELECT count(Name) FROM mysql.time_zone_name" \
+	| ( docker run --rm -i \
+	--link "$cname":mysql \
+	--entrypoint mysql \
+	-e MYSQL_PWD="$MYSQL_ROOT_PASSWORD" \
+	"$image" \
+	-hmysql \
+	-u"root" \
+	--silent \
+	"mysql" \
+	2>&1 || : ) \
+	| grep -q "^0$"
```

Comment:
My comment here is similar to my comment on `mysql-skip-resolve` -- does this need to be an explicit separate test?  Also, why is this one so much more complex than that one?  Shouldn't it be a similar `[ "$(echo ... | mysql)" = '0' ]`?

Reply:
the goal of this test - check if `MYSQL_INITDB_SKIP_TZINFO` variable works fine, so it is needed separate test.
I have added a check into `mysql-basics`which verify that tzinfo is imported by default.
this test - checks if tzinfo is not imported.
anyway, reworked test in `[ "$(echo ... | mysql)" = '0' ]` way